Filtering.md

```diff
@@ -159,3 +159,20 @@ end
 ```
 
 These declarations can also be overridden from the command line.
+
+## Silencing filter announcements
+
+By default, RSpec will print a message before your specs run indicating what
+filters are configured, for instance, it might print
+"Run options: include {:focus=>true}" if you set `config.filter_run_including
+:focus => true`.
```

Comment:
Relish will insert line breaks at the same places you have it here which won't look good.  We're moving in the direction of having each paragraph be one lone unwrapped line so that HTML flow can wrap it.  That looks much better.  Can you change this to one long line?